.ci/docker-compose-file/docker-compose-emqx-cluster-rlog.override.yaml

```diff
@@ -23,6 +24,9 @@ services:
   emqx2:
     <<: *default-emqx
     container_name: node2.emqx.io
+    depends_on:
+      - emqx1
+    restart: on-failure
```

Comment:
I had to add those to make the startup more robust.  Occasionally, there's some kind of race condition with the timing of startup of the node that makes one (or both) crash while trying to connect to each other.